alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -179,6 +179,7 @@ private function getAutoEmbedCode($targetId = null)
 
 		$config["partnerId"] = $this->partnerId;		
 		$config["uiConfId"] = $this->uiconfId;
+		$config["plugins"] = $this->playerPlugins;
```

Comment:
This needs to be added also to dynamic embed

Reply:
done